tests/test_auth.py

```diff
@@ -0,0 +1,35 @@
+from httpx import Cookies
+
+from app import models
+from app.auth_manager import get_strategy
+from app.config import settings
+from app.utils.enums import RequestMethod
+from tests.utils import make_http_request
+
+
+async def test_refresh_token_handling(test_user: models.User):
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (testing):** Add test cases for expired refresh tokens.

It's important to verify the system's behavior when an expired refresh token is used, ensuring it handles such scenarios gracefully.

```suggestion
async def test_refresh_token_handling_expired(test_user: models.User):
    """
    Test function to verify the system's behavior when an expired refresh token is used.

    Args:
        test_user: The test user object for token refresh testing.

    Returns:
        None
    """

    strategy = get_strategy()
    expired_token = await strategy.write_refresh_token(test_user, expired=True)

    cookies = Cookies(
        {
            settings.refresh_token_name: expired_token,
        }
    )

    endpoint = "/api/users/me"

    res = await make_http_request(
        url=endpoint, method=RequestMethod.GET, cookies=cookies
    )
    assert res.status_code == 401
```